Projects/1.Counter/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Counter</title>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+
+  <body>
+    <div class="div">
+      Counter
+      <br />
+      <br />
+      <p class="p">
+        <a id="clicks">0</a>
+      </p>
+      <br />
+      <p class="pointer Increase" onclick="ICounter();">Add Count</p>
```

Comment:
Write class name as  `pointer-increase` we mostly use kebab case. https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ Go through this once.